fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/impl/IndexBuilderImpl.java

```diff
@@ -148,6 +166,13 @@ private void indexOcflObject(final String ocflId, final String txId, final OcflO
                                 fedoraId.getFullId()));
                     }
                 }
+                if (!headers.getInteractionModel().equalsIgnoreCase(NON_RDF_SOURCE.toString())) {
```

Comment:
Doesn't seem like we need a case insensitive comparison here, the interaction model uris can be case sensitive.